hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestHiveSyncTool.java

```diff
@@ -189,7 +189,7 @@ private static Iterable<Object[]> syncDataSourceTableParams() {
     return opts;
   }
 
-  @BeforeEach
+  @BeforeAll
```

Comment:
This test class contains 180+ test methods, which may cause dramatical resource leak. Most likely this is the main source of flakiness of the module.